storey/drivers.py

```diff
@@ -501,3 +509,450 @@ async def _get_item(self, container, table_path, key, attributes):
+                        array_time_attribute_key = self._aggregation_time_key(redis_key_prefix, aggr_time_attribute_name)
+
+                        cached_time = bucket.storage_specific_cache.get(array_time_attribute_key, 0)
+
+                        expected_time = int(bucket_start_time / bucket.max_window_millis) * bucket.max_window_millis
+                        expected_time_expr = self._convert_python_obj_to_lua_value(
+                            datetime.fromtimestamp(expected_time / 1000))
+                        index_to_update = int((bucket_start_time - expected_time) / bucket.period_millis)
+
+                        for aggregation, aggregation_value in aggregation_values.items():
+                            list_attribute_name = f'{self._aggregation_attribute_prefix}{name}_{aggregation}_{feature_attr}'
+                            list_attribute_key = self._list_key(redis_key_prefix, list_attribute_name)
+                            if list_attribute_key not in redis_keys_involved:
+                                redis_keys_involved.append(list_attribute_key)
+                            lua_script = f'{lua_script}list_attribute_key="{list_attribute_key}";\n'
+
+                            if cached_time < expected_time:
+                                if not initialized_attributes.get(list_attribute_key, 0) == expected_time:
+                                    initialized_attributes[list_attribute_key] = expected_time
+                                    lua_script = f'{lua_script}local t=redis.call("GET","{array_time_attribute_key}");if (type(t)~="boolean" and (tonumber(t) < {expected_time})) \
+                                        then redis.call("DEL",list_attribute_key); end;\n'
+                                    default_value = self._convert_python_obj_to_redis_value(aggregation_value.default_value)
+                                    lua_script = f'{lua_script}len=redis.call("LLEN",list_attribute_key) for i=1,({bucket.total_number_of_buckets}-len) \
+                                        do redis.call("RPUSH",list_attribute_key,{default_value}) end;\n'
+                                if array_time_attribute_key not in times_updates:
+                                    times_updates[array_time_attribute_key] = expected_time_expr
+                                new_cached_times[name] = (array_time_attribute_key, expected_time)
+
+                            # Updating the specific cells
+                            if cached_time <= expected_time:
+                                # condition = None
+                                # if partitioned_by_key and self._mtime_name in aggregation_element.storage_specific_cache:
+                                #     condition = aggregation_element.storage_specific_cache[self._mtime_name]
+                                lua_script = f'{lua_script}local old_value=redis.call("LINDEX", list_attribute_key, {index_to_update});\
```

Comment:
I think it's better to settle on no space after commas, or with space after commas, but not a mixture of the two.

Reply:
Agree

Reply:
fixed